nir/__init__.py

```diff
@@ -15,6 +15,7 @@
 from . import ir
 from .ir import *  # noqa: F403
 from .ir import typing  # noqa: F401
+from .data_ir import *  # noqa: F403
```

Comment:
We should also add it to the `__all__` list, so it features when people import nir.

`__all__ = ir.__all__ + data_ir.__all__ + ["read", "write"]` should do it ( if `__all__` is defined in `data_ir/__init__.py`)